earthquakes.py

```diff
@@ -1,61 +1,116 @@
             "minlongitude": "-9.756",
             "minmagnitude": "1",
-            "endtime": "2018-10-11",
+            "endtime": end_time,
             "orderby": "time-asc"}
     )
 
+
     # The response we get back is an object with several fields.
     # The actual contents we care about are in its text field:
     text = response.text
     # To understand the structure of this text, you may want to save it
     # to a file and open it in VS Code or a browser.
     # See the README file for more information.
-    ...
 
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    return json.loads(text)
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
+    return data["metadata"]["count"]
 
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
+    return earthquake["properties"]["mag"]
 
 
 def get_location(earthquake):
     """Retrieve the latitude and longitude of an earthquake item."""
+    coordinates = earthquake["geometry"]["coordinates"]
     # There are three coordinates, but we don't care about the third (altitude)
-    return ...
+    return (coordinates[0], coordinates[1])
 
 
 def get_maximum(data):
     """Get the magnitude and location of the strongest earthquake in the data."""
-    ...
+    current_max_magnitude = get_magnitude(data["features"][0])
+    current_max_location = get_location(data["features"][0])
+    for item in data["features"]:
+        magnitude = get_magnitude(item)
+        # Note: what happens if there are two earthquakes with the same magnitude?
+        if magnitude > current_max_magnitude:
+            current_max_magnitude = magnitude
+            current_max_location = get_location(item)
+    return current_max_magnitude, current_max_location
+    # There are other ways of doing this too:
+    # biggest_earthquake = sorted(data["features"], key=get_magnitude)[0]
+    # return get_magnitude(biggest_earthquake), get_location(biggest_earthquake)
+    # Or...
+    # biggest_earthquake = max(
+    #     ({"mag": get_magnitude(item), "location": get_location(item)}
+    #     for item in data["features"]),
+    #     key=lambda x: x["mag"]
+    # )
+    # return biggest_earthquake["mag"], biggest_earthquake["location"]
+
+n_earthquakes = np.zeros(21)
+n_avarage = np.zeros(21)
+
+for i in range(2000,2021):
+    start_time = str(i) + '-01-01'
+    end_time = str(i+1) + '-01-01'
+    data = get_data(start_time,end_time)
+    n_earthquakes[i-2000] = count_earthquakes(data)
+    sum = 0
+    for j in range(count_earthquakes(data)):
+        sum = sum + get_magnitude(data["features"][j])
+    n_avarage[i-2000] = sum/(count_earthquakes(data)+1)
+
+'''
+plt.plot(np.arange(2000,2021,1),n_earthquakes)
+plt.xlabel('Year')
+plt.ylabel('Number of earthquakes')
+plt.savefig('num_earthquakes_year')
+'''
```

Comment:
In general, the multi-line comments like this are used for documenting functions etc (docstrings). If you need to comment out many lines of code, use `#` on each line. Many editors have a shortcut for doing that on multiple lines at once, e.g. `Ctrl + /` or `Cmd + /` in VS Code. But also think about whether you really need to comment it out or not - it can lead to confusion when rereading the code.